client/src/components/pages/MainChannel.tsx

```diff
@@ -93,6 +107,59 @@ interface ChannelProps {
   searchArgs: ChannelsQueryVariables;
 }
 
+const ModalContainer = styled.View`
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+`;
+
+const ModalViewContainer = styled.View`
+  width: 80%;
+  height: 25%;
```

Comment:
Does this look okay on web?

Reply:
As a result of checking, this modal some problem in web. so i'll fix this. thank you 😄